### ISSUE TITLE
Running processes and cpu usage not showing 

### ISSUE THREAD

**Issue by SucubusJenny** [webmin/webmin#2242]
After just installing Webmin on alpine Linux I am not able to get running processes or CPU usage to show up.
![Screenshot_20240730-010058](https://github.com/user-attachments/assets/09f91d6f-d754-4a41-93bd-6252ebd5fc5e)

**Comment by SucubusJenny** [comment]
log file seems to indicate an issue with PS

`[30/Jul/2024:03:25:25 -0400] Reloading configuration
	-T			Show threads
	-o COL1,COL2=HEADER	Select columns for display

Show list of processes

Usage: ps [-o COL1,COL2=HEADER] [-T]

BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.
ps: unrecognized option: x
sysctl: error: 'hw/ncpu' is an unknown key
sysctl: error: 'hw/model' is an unknown key
	-T			Show threads
	-o COL1,COL2=HEADER	Select columns for display

Show list of processes

Usage: ps [-o COL1,COL2=HEADER] [-T]

BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.
ps: unrecognized option: x
	-T			Show threads
	-o COL1,COL2=HEADER	Select columns for display

Show list of processes

Usage: ps [-o COL1,COL2=HEADER] [-T]

BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.
ps: unrecognized option: x
	-T			Show threads
	-o COL1,COL2=HEADER	Select columns for display

Show list of processes

Usage: ps [-o COL1,COL2=HEADER] [-T]

BusyBox v1.36.1 (2024-06-10 07:11:47 UTC) multi-call binary.
ps: unrecognized option: x
sysctl: error: 'hw/ncpu' is an unknown key
sysctl: error: 'hw/model' is an unknown key
[30/Jul/2024:02:20:58 -0400] Using SHA512 module Crypt::SHA
[30/Jul/2024:02:20:58 -0400] Using MD5 module Digest::MD5
[30/Jul/2024:02:20:58 -0400] IPv6 support cannot be enabled without the Socket6 perl module
[30/Jul/2024:02:20:58 -0400] miniserv.pl started
[30/Jul/2024:02:20:54 -0400] Shutting down
[29/Jul/2024:23:46:00 -0400] [192.168.1.12] /sysinfo.cgi?no-cache=1 : The provided data is not a valid EOL data hash reference
[29/Jul/2024:23:45:35 -0400] Using SHA512 module Crypt::SHA
[29/Jul/2024:23:45:35 -0400] Using MD5 module Digest::MD5
[29/Jul/2024:23:45:35 -0400] IPv6 support cannot be enabled without the Socket6 perl module
[29/Jul/2024:23:45:35 -0400] miniserv.pl started
[29/Jul/2024:23:45:33 -0400] Restarting
[29/Jul/2024:23:45:04 -0400] Using SHA512 module Crypt::SHA
[29/Jul/2024:23:45:04 -0400] Using MD5 module Digest::MD5
[29/Jul/2024:23:45:04 -0400] IPv6 support cannot be enabled without the Socket6 perl module
[29/Jul/2024:23:45:04 -0400] miniserv.pl started
[29/Jul/2024:23:45:00 -0400] Restarting
10485760 bytes (10.0MB) copied, 0.040568 seconds, 246.5MB/s`

**Comment by jcameron** [comment]
Which Linux distribution and version are you running Webmin on there?

**Comment by SucubusJenny** [comment]
Running Webmin on alpine 3.20

**Comment by jcameron** [comment]
AH ...that's not a supported distribution unfortunately. Is it based on some other more common distro, like Debian?

**Comment by SucubusJenny** [comment]
I was able to fix the processes not showing by installing procps. but after doing so the cpu usage graph does not appear to work. it appears that webmin requires to use PS -x which is not supported on the alpine default ps but does work with procps.

**Comment by jcameron** [comment]
Ok looks like we are already discussing detection of Alpine properly by Webmin in https://github.com/webmin/webmin/issues/1312

**Comment by mckaygerhard** [comment]
no! .. i already did in my previous comment .. @jcameron  please check #2353 now you can close this cos was addressed there!

**Comment by mckaygerhard** [comment]
now confirms cpu usage is working with my guide, the cpu stats are too low so graph is not show due low rate..  picture now shows some activity.. for Alpine support status check #2353  this can be close!

![image](https://github.com/user-attachments/assets/63261037-2841-49e9-a8db-32fc392f8b58)
